http/rest-client/src/main/java/io/quarkus/ts/http/restclient/GzipResource.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.ts.http.restclient;
```

Comment:
This scenario should be located in http-advanced module, rest-client is not leveraged at all and it is causing problems in the end because of bad use.